### PR TITLE
Use consistent page titles across Insights - add subapp name to page titles

### PR DESCRIPTION
**Pull request by mhuth**
Not sure if this is the right/react way of doing this.  Feedback is most welcome.

Old issue: https://issues.redhat.com/browse/ADVISOR-608
Mocks: https://marvelapp.com/prototype/e5gcj47/screen/71812909

Screenshots of changes:
recommendations_title
systems_title
topics_title

### CODE REVIEW

**Review by AllenBW — APPROVED** [review]
I like this! this is goooooodddddddd <3  nicely done @mhuth  🤗

File: src/Messages.js [hunk 1]
```diff
@@ -2,6 +2,11 @@
 import { defineMessages } from 'react-intl';
 
 export default defineMessages({
+    documentTitle: {
+        id: 'documentTitle',
+        description: 'The title of the page as it appears in the browser tab',
+        defaultMessage: 'Advisor {subapp} | Red Hat Insights'
```
Comment (ryelo) [inline src/Messages.js:8]: This should actually be '{subapp} - Advisor | Red Hat Insights'

The `subapp` should also be capitalized. ex) `Topics - Advisor | Red Hat Insights`

**Review by ryelo — CHANGES_REQUESTED** [review]

**Review by AllenBW — CHANGES_REQUESTED** [review]
OK i still love this, but looks like we gotta go ah step further @mhuth https://docs.google.com/presentation/d/19voFhv_MUGwp-DNRj59mK_2YrignJLSWF9Jvbxk2bSU/edit#slide=id.g8e531b9fb1_0_0

need those detail pages to update the title as well, for example, from link above,`SAP - Topics - Advisor | Red Hat Insights`

**Review by terezanovotna — APPROVED** [review]

**Comment by terezanovotna** [comment]
Looking good - fast turnaround on this!

**Comment by mhuth** [comment]
Change page title to format `SubNav - Advisor | Red Hat Insights` and pre-pended topic names as required.

topics
sap_topic
mssql_topic

**Comment by AllenBW** [comment]
Awesome quick additions! gotta wonder, doc says to mirror breadcrumbs... probably means we should also put rec details up there (the rec name)... wonder if that means we should put system name up there? (thinkin not, that gets long)

**Comment by mhuth** [comment]
As per the breadcrumbs, I've prepended the rule description and system display_name to the document title.  The rule description can make the title very long.  The system display_name isn't too bad.  See screenshots ...

Reasonable rule description ...
recs1

Very long rule description ...
recs2

System display_name ...
systems1

System display_name in
systems2
 document title from the recommendation system list ...

**Review by AllenBW — APPROVED** [review]
Looks good to me! nice work @mhuth 🤗 👍

**Pull request merged**